src/main/java/edu/utdallas/davisbase/parser/Parser.java

```diff
@@ -15,13 +34,204 @@ public Parser(ParserConfiguration configuration) {
+          selectStatement.toString(),
+          splitStr[2],
+          Arrays.asList(splitValues),
+          splitValues[0].equalsIgnoreCase("*") ? true : false
+        );
+        return select;
+      }
+      else{
+        throw new ParseException("Sorry DavisBase does not support this command");
+      }
+    }
+    catch(JSQLParserException e){
+      throw(new ParseException(e.getCause()));
+    }
+    catch(ParseWhereException e){
+      throw(new ParseException(e.getCause()));
+    }
   }
 
+  /**
+   * @param where clause to parse
+   * @return WhereExpression representation of the expression
+   */
+  public WhereExpression parseWhereExpression(Expression where) throws ParseWhereException{
+    WhereExpression whereExpression;
+    if(where instanceof EqualsTo){
+      EqualsTo equals = (EqualsTo) where;
+      whereExpression= new WhereExpression(
+        equals.toString(),
+        equals.isNot(),
+        equals.getLeftExpression(),
+        equals.getStringExpression(),
+        equals.getRightExpression()
+      );
+      return whereExpression;
+    }
+    else if(where instanceof NotEqualsTo){
+      NotEqualsTo notEqualsTo = (NotEqualsTo) where;
+      whereExpression= new WhereExpression(
+        notEqualsTo.toString(),
+        notEqualsTo.isNot(),
+        notEqualsTo.getLeftExpression(),
+        notEqualsTo.getStringExpression(),
+        notEqualsTo.getRightExpression()
+      );
+      return whereExpression;
+    }
+    else if(where instanceof GreaterThan){
+      GreaterThan greaterThan = (GreaterThan)where;
+      whereExpression= new WhereExpression(
+        greaterThan.toString(),
+        greaterThan.isNot(),
+        greaterThan.getLeftExpression(),
+        greaterThan.getStringExpression(),
+        greaterThan.getRightExpression()
+      );
+      return whereExpression;
+    }
+    else if(where instanceof GreaterThanEquals){
+      GreaterThanEquals greaterThanEquals = (GreaterThanEquals)where;
+      whereExpression= new WhereExpression(
+        greaterThanEquals.toString(),
+        greaterThanEquals.isNot(),
+        greaterThanEquals.getLeftExpression(),
+        greaterThanEquals.getStringExpression(),
+        greaterThanEquals.getRightExpression()
+      );
+      return whereExpression;
+    }
+    else if(where instanceof MinorThan){
+      MinorThan minorThan = (MinorThan)where;
+      whereExpression= new WhereExpression(
+        minorThan.toString(),
+        minorThan.isNot(),
+        minorThan.getLeftExpression(),
+        minorThan.getStringExpression(),
+        minorThan.getRightExpression()
+      );
+      return whereExpression;
+    }
+    else if(where instanceof MinorThanEquals){
+      MinorThanEquals minorThanEquals = (MinorThanEquals)where;
+      whereExpression= new WhereExpression(
+        minorThanEquals.toString(),
+        minorThanEquals.isNot(),
+        minorThanEquals.getLeftExpression(),
+        minorThanEquals.getStringExpression(),
```

Comment:
Let's either pass the already-structured `MinorThanEquals` object or--better yet--use some sort of our own structured enum (e.g. `...representation.ComparisonOperator.MINOR_THAN_EQUALS`) rather than a raw string for this.